micromouse/TeensyFunctions.h

```diff
@@ -1,9 +1,65 @@
-#ifdef  #ifdef __MK20DX256__ // Teensy Compile
+#ifndef TEENSYFUNCTIONS_H
+#define TEENSYFUNCTIONS_H
+
+#include "ButtonFlag.h"
+//Teensy PC safe versions
+//Controller.cpp
+
+extern volatile bool buttonFlag;
+
+#ifdef __MK20DX256__ // Teensy Compile
 #else //on windows
-void delay(int milliseconds){};
+void delay(unsigned long milliseconds){};
```

Comment:
the functions actually need to be static as well to prevent any linker problems during compiling

`static void delay(unsigned long milliseconds){};`

all of them need to be static